glean-core/python/setup.py

```diff
@@ -177,14 +177,23 @@ def run(self):
         if target == "i686-pc-windows-gnu":
             env["RUSTFLAGS"] = env.get("RUSTFLAGS", "") + " -C panic=abort"
 
-        command = ["cargo", "build", "--package", "glean-ffi", "--target", target]
+        command = [
+            "cargo",
+            "build",
+            "--package",
+            "glean-ffi",
+            "--target",
+            target,
+            "--features",
+            "rkv-safe-mode",
+        ]
         if buildvariant != "debug":
             command.append(f"--{buildvariant}")
 
         if "-darwin" in target:
             env["MACOSX_DEPLOYMENT_TARGET"] = macos_compat(target)
 
-        subprocess.run(command, cwd=SRC_ROOT, env=env)
+        subprocess.check_call(command, cwd=SRC_ROOT / "glean-core", env=env)
```

Comment:
Why do we change the working directory now?

Reply:
I got this -->

```
error: --features is not allowed in the root of a virtual workspace
note: while this was previously accepted, it didn't actually do anything
help: change the current directory to the package directory, or use the --manifest-path flag to the path of the package
```

Reply:
Oh right, but then we should switch directly to the `glean-core/ffi` directory. Not even sure why it works in `glean-core`. 
Weird interaction of cargo to not allow features in the workspace root, but then somehow in a subdirectory that's not even the final one.

... which now brings me to another point: `rkv-safe-mode` is a feature of `glean-core`, not of `glean-ffi`. We need to re-expose that on `glean-ffi` to actually enable it on the build, I think.
While we're at it `glean` (RLB) should also get it, because right now we need to explicitely reference `glean-core` in FOG.

I can do that tomorrow, before we than update this PR.

Reply:
Updated this here (and rebased to get the feature delegation changes).